cpp/velox/memory/VeloxMemoryManager.cc

```diff
@@ -162,41 +162,52 @@ class ListenableArbitrator : public velox::memory::MemoryArbitrator {
   gluten::AllocationListener* listener_;
 };
 
-VeloxMemoryManager::VeloxMemoryManager(
-    std::string name,
+velox::memory::IMemoryManager::Options VeloxMemoryManager::getOptions(
     std::shared_ptr<MemoryAllocator> allocator,
-    std::shared_ptr<AllocationListener> listener)
-    : MemoryManager(), name_(name), listener_(std::move(listener)) {
+    std::shared_ptr<AllocationListener> listener) const {
```

Comment:
If use `listener_` to construct ListenableArbitrator, why pass a unused `listener`?